package.json

```diff
@@ -38,53 +38,69 @@
     "url": "git+https://github.com/catarak/p5.js-web-editor.git"
   },
   "devDependencies": {
-    "babel-eslint": "^7.1.1",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-transform-react-constant-elements": "^6.23.0",
-    "babel-plugin-transform-react-inline-elements": "^6.22.0",
+    "@babel/core": "^7.0.0",
```

Comment:
This is listed in `devDependencies` and `dependencies`.

Reply:
weird! i used `npm babel-upgrade` to move all of the packages, which must have saved it in both. i'll move it to just `dependencies`